internal/deploy/bootstrap.go

```diff
@@ -70,6 +71,7 @@ func Bootstrap(
 	if err != nil {
 		return false, err
 	}
+	manifestObjs = filterCRD(manifestObjs)
```

Comment:
`filter` functions are usually used to return a subset of some collection. Here we're modifying objects. Perhaps `fixTimestamps` or something similar is a better choice.